src/common/components/cards/how-to-check-card-row.tsx

```diff
@@ -1,24 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { HowToCheckText } from 'common/components/cards/how-to-check-text';
 import * as React from 'react';
 import { CardRowProps } from '../../../common/configs/unified-result-property-configurations';
 import { NamedFC } from '../../../common/react/named-fc';
+import * as styles from './how-to-check-card-row.scss';
 import { SimpleCardRow } from './simple-card-row';
 
 export interface HowToCheckWebPropertyData {
-    checkText: string;
+    ruleId: string;
 }
 
 export interface HowToCheckWebCardRowProps extends CardRowProps {
     propertyData: HowToCheckWebPropertyData;
 }
+
 export const HowToCheckWebCardRow = NamedFC<CardRowProps>(
     'HowToCheckWebCardRow',
     ({ ...props }) => {
+        const getText = () => {
```

Comment:
You don't have make this a function: you should be able to just have the HowToCheckText component inline.

Reply:
If you're worried about too much nesting, you can also just set a variable locally that stores whatever you're putting into content as well.

Reply:
done!